internal/server/data/migrations.go

```diff
@@ -923,3 +924,19 @@ func modifyAccessKeysIndex() *migrator.Migration {
 		},
 	}
 }
+
+func moveAllowedDomainsToOrganizationsTable() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "2022-11-21T11:00",
+		Migrate: func(tx migrator.DB) error {
+			stmt := `ALTER TABLE providers DROP COLUMN IF EXISTS allowed_domains`
+			if _, err := tx.Exec(stmt); err != nil {
+				return err
+			}
+
+			stmt = `ALTER TABLE organizations ADD COLUMN IF NOT EXISTS allowed_login_domains text DEFAULT ''`
```

Comment:
Not adding any allowed social login domains in this migration, we can expose it as a setting in the UI for existing orgs once other social login changes are in.